File: Python Excel Examples/CellsApiDemo/cell/getCells.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration
from spirecloudexcel.api.cells_api import CellsApi

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.cells_api.CellsApi(configuration)

name = "GetCells_1.xlsx"
folder = "ExcelDocument"
storage = ""
sheetName = "Sheet1"
offset = 0
count = 0
result = api.get_cells(name, sheetName, offest=offset, count=count, folder=folder, storage=storage)

File: PHP Excel Examples/CellsApiDemo/Cell/getCells.php
<?php

use Spire\Cloud\Excel\Sdk\Api\CellsApi;
use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new CellsApi($configuration);

$name = "GetCells.xlsx";
$folder = "input";
$storage = null;
$sheetName = "Sheet1";
$offest = 0;
$count = 0;
$result = $apiInstance->getCells($name, $sheetName, $offest, $count, $folder, $storage);
